src/components/core/geoChart/legends/PushpinSizeLegend.tsx

```diff
@@ -0,0 +1,33 @@
+// (C) 2020 GoodData Corporation
+import * as React from "react";
+import { formatLegendLabel } from "../../../../components/visualizations/utils/common";
+
+export interface IPushpinSizeLegendProps {
+    format: string;
+    numericSymbols: string[];
+    sizeValues: [number, number, number];
```

Comment:
it could be just `sizes: number[]`
All calculation logic should happen in `PushpinSizeLegend` itself, like calculating min, max and avg

Reply:
fixed